tests/suites/test_suite_psa_crypto_metadata.function

```diff
@@ -58,15 +58,15 @@
 
 /* Check the parity of value.
  * Return 0 if value has even parity and a nonzero value otherwise. */
-int test_parity( uint32_t value )
+int check_parity( uint32_t value )
```

Comment:
Function description above:
`Check the parity of value is even.` rather than just `Check the parity of value.` ?

Reply:
The semantics of this function is to check that the parity is the one that's built into PSA crypto constants. The fact that this is even parity is just a detail.

This isn't the first time that it's come up: https://github.com/ARMmbed/mbed-crypto/pull/330#discussion_r360305364 (where I learned that some people would say that this function check that the parity is odd). I guess I should make the comment more explicit. I'll do that.

Reply:
The return value convention is the opposite of our usual convention for `check_` functions: usually 0 means the value was found to be acceptable, and non-zero that it wasn't. So I'd expect each line that starts with `TEST_ASSERT( check_` to end with `) == 0 )` but the definition of `TEST_PARITY()` below doesn't.

Reply:
Thanks for adding comments, much better with them.